feature/gnoi/os/tests/osinstall/osinstall_test.go

```diff
@@ -94,6 +94,9 @@ func TestOSInstall(t *testing.T) {
 		tc.activateOS(ctx, t, true)
 	}
 	tc.rebootDUT(ctx, t)
+	if tc.dualSup {
```

Comment:
tc.rebootDUT already waited 10min, and we are adding possible another 15min on top of that. Any optimization we can do here?

Also, we only improved the dualSup case, what about single Sup case? Any thing we can optimize there too? 

I feel like that either we brutal force update the rebootDUT to wait 5 more min, or we do something smarter for **both** dualSup and singleSup cases.

Reply:
tc.rebootDUT waits for the chassis to be rebooted, but there seemed to be issues when the standby supervisor was rebooting, there was no case to handle certain states in those

I haven't fully tested this against singleSup devices, yet. I think this method is more comprehensible for dualSup than simply adding a 'X' Minute timer to rebootDUT, no?

Reply:
my bad. I was looking at the wrong code.